src/DonorProfiles/resources/js/app/components/auth-modal/index.js

```diff
@@ -72,60 +72,60 @@ const AuthModal = () => {
 	};
 
 	return (
-		<div className="give-donor-profile__auth-modal">
-			<div className="give-donor-profile__auth-modal-frame">
-				<div className="give-donor-profile__auth-modal-heading">
+		<div className="give-donor-dashboard__auth-modal">
+			<div className="give-donor-dashboard__auth-modal-frame">
+				<div className="give-donor-dashboard__auth-modal-heading">
 					{ __( 'Log in to see your donor dashboard', 'give' ) }
 				</div>
-				<div className="give-donor-profile__auth-modal-content">
+				<div className="give-donor-dashboard__auth-modal-content">
 					{ loggedInWithoutDonor && (
-						<div className="give-donor-profile__auth-modal-notice">
-							{ __( 'The account you are currently logged into the site with does not have an associated donor profile. Donate now or contact the site administrator to associate this WordPress user with a donor profile.' ) }
+						<div className="give-donor-dashboard__auth-modal-notice">
+							{ __( 'The account you are currently logged into the site with does not have an associated donor dashboard. Donate now or contact the site administrator to associate this WordPress user with a donor dashboard.' ) }
```

Comment:
Can we add the missing text domain?

Reply:
Done, ref: https://github.com/impress-org/givewp/pull/5708/commits/4fb4d2daea978aa6b2006a3ea5459febfd146784